account_invoice_report_grouped_by_picking/models/account_move.py

```diff
@@ -108,6 +108,10 @@ def lines_grouped_by_picking(self):
                         )
                         picking_dict.setdefault(key, 0)
                         qty = so_line.product_uom_qty
+                        if remaining_qty < so_line.product_uom_qty:
+                            qty = so_line.product_uom_qty - (
+                                so_line.product_uom_qty - remaining_qty
+                            )
```

Comment:
Matemáticamente, esto es lo mismo:

```suggestion
                            qty = remaining_qty
```

Reply:
done